### ISSUE TITLE
[MDX] Line break between JSX tags

### ISSUE THREAD

**Issue by wonskarol** [prettier/prettier#7710]
**Prettier 1.19.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAwhAJnAogDwEMBbABwBs4A+AHSgAJ7UBBegelqlTcxwJIrU6dDNjxEylTo1QAhdp269xAqSAA0ICKRgBLaAGdkoQgCdTEAO4AFMwiMpCANwi6sGkACNThMAGs4GABlUl9dKABzZBhTAFc4TQALGGJyAHVE3XgDULA4IPss3ScsgE9kcAMjTXCDOFMYax8I4kJkADNCcjrNACsDfFkff0CgkjgAGXC4Dq6ekH78IPCIygBFWIh4We6EkFDTOtMK4ix8D1JTcJg0txhE5AAOAAZNS4g6tJ9SCsu4I6cM00AEdNvAmtoHCBCAYALRQOBwHDuTSmOCg3RopqEFptJCdXaaOrEXTROJ7AwrdZgmb4uZ7GCETy3LD3ZAAJk0MUIunIK0wxFaFSg0CBIFidQAKkyHAT5k54gBJKA4WBBMBXHTMFVBGClSg7OoAXyNQA)
```sh
--parser mdx
```

**Input:**
```mdx
<CodeExample>
  <A />
</CodeExample>

<CodeExample>
  <B />
</CodeExample>
```

**Output:**
```mdx
<CodeExample>
  <A />
</CodeExample>
<CodeExample>
  <B />
</CodeExample>

```

**Expected behavior:**

Prettier should preserve new line between JSX tags because there is a difference for MDX parser:

1) 
```mdx
<CodeExample>
  <A />
</CodeExample>

<CodeExample>
  <B />
</CodeExample>
```
 produces **two MDAST children nodes**:
```
{
  "type": "root",
  "children": [
    {
      "type": "jsx",
      "value": "<CodeExample>\n  <A />\n</CodeExample>",
      "position": {
        "start": {
          "line": 1,
          "column": 1,
          "offset": 0
        },
        "end": {
          "line": 3,
          "column": 15,
          "offset": 36
        },
        "indent": [
          1,
          1
        ]
      }
    },
    {
      "type": "jsx",
      "value": "<CodeExample>\n  <B />\n</CodeExample>",
      "position": {
        "start": {
          "line": 5,
          "column": 1,
          "offset": 38
        },
        "end": {
          "line": 7,
          "column": 15,
          "offset": 74
        },
        "indent": [
          1,
          1
        ]
      }
    }
  ],
  "position": {
    "start": {
      "line": 1,
      "column": 1,
      "offset": 0
    },
    "end": {
      "line": 7,
      "column": 15,
      "offset": 74
    }
  }
}
```

2) 
```mdx
<CodeExample>
  <A />
</CodeExample>
<CodeExample>
  <B />
</CodeExample>
```
 produces **one MDAST child node**:
```
{
  "type": "root",
  "children": [
    {
      "type": "jsx",
      "value": "<CodeExample>\n  <A />\n</CodeExample>\n<CodeExample>\n  <B />\n</CodeExample>",
      "position": {
        "start": {
          "line": 1,
          "column": 1,
          "offset": 0
        },
        "end": {
          "line": 6,
          "column": 15,
          "offset": 73
        },
        "indent": [
          1,
          1,
          1,
          1,
          1
        ]
      }
    }
  ],
  "position": {
    "start": {
      "line": 1,
      "column": 1,
      "offset": 0
    },
    "end": {
      "line": 6,
      "column": 15,
      "offset": 73
    }
  }
}
```

This behaviour can have bad consequences when you use eg. remark plugins.

**Comment by thorn0** [comment]
This seems to have been fixed in 2.0.2.

@wonskarol Please confirm it works now.

**Comment by wonskarol** [comment]
@thorn0 it works!  Thanks for update.
